src/muler/echelle.py

```diff
@@ -524,3 +536,35 @@ def plot(self, **kwargs):
             self[i].plot(ax=ax, **kwargs)
 
         return ax
+
+    def __add__(self, other):
+        """Bandmath addition
+        """
+        spec_out = copy.deepcopy(self)
+        for i in range(len(self)):
+            spec_out[i] = self[i] + other[i]
```

Comment:
This pattern should work!
In the future we may want to test whether the metadata and uncertainties are automatically passed along.  We can make it explicit by saying `spec1.add(spec2, handle_meta='first_found', propagate_uncertainty=True)` rather than `spec1 + spec2` and so on for multiply, divide and subtract.  

We may also want to add some checking for whether the spectral axes are aligned.  If not, bandmath gets weird, and the specutils default may be unexpected.

Reply:
I tested basic band math and it seems to work.  Propagation of uncertainty and other things are also one of my concerns for specutils (does it automatically do it if a spectrum object has uncertainty?).  As for the wavelength solutions being misaligned, does specutils do some kind of interpolation to align them?

Reply:
See this tutorial for the answers to your questions--- it's a bit subtle how specutils handles misaligned spectra. https://muler.readthedocs.io/en/latest/tutorials/Combining_uncertainties_with_specutils.html